npm/webpack-dev-server/src/helpers/vueCliHandler.ts

```diff
@@ -5,21 +5,23 @@ import { sourceDefaultWebpackDependencies } from './sourceRelativeWebpackModules
 
 const debug = debugLib('cypress:webpack-dev-server:vueCliHandler')
 
-export function vueCliHandler (devServerConfig: WebpackDevServerConfig): PresetHandlerResult {
+export async function vueCliHandler (devServerConfig: WebpackDevServerConfig): Promise<PresetHandlerResult> {
   const sourceWebpackModulesResult = sourceDefaultWebpackDependencies(devServerConfig)
 
   try {
-    const config = require.resolve('@vue/cli-service/webpack.config', {
+    const Service = require(require.resolve('@vue/cli-service', {
```

Comment:
@msebas realized that the `main` file of `@vue/cli-service` is the Service file so we don't need a deep import to resolve it. Also, I pushed up a test!